### ISSUE TITLE
Add option to disable bell

### ISSUE THREAD

**Issue by nburns** [dragonworx/grunt-bell#1]
It would be really nice if grunt-bell could be disabled with a flag, or if it would respect grunts own bell settings. gruntjs/grunt-contrib-watch/issues/93

**Comment by dragonworx** [comment]
Good idea. I will work on it soon and update issue when ready. Cheers.
